changelogs/fragments/705-gitlab-auth-support.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - gitlab - add authentication support using oauth_token and job_token (https://github.com/ansible-collections/community.general/issues/705).
```

I think it's better to mention the names of the new options:
```suggestion
  - gitlab - add more token authentication support with the new options ``api_oauth_token`` and ``api_job_token`` (https://github.com/ansible-collections/community.general/issues/705).
```